docs-ui/stories/components/multipleCheckbox.stories.js

```diff
@@ -6,20 +7,30 @@ export default {
   title: 'Components/Forms/Controls/Multiple Checkbox',
   component: MultipleCheckbox,
   args: {
-    choices: [
-      ['foo', 'Foo'],
-      ['bar', 'Bar'],
-      ['baz', 'Baz'],
-      ['quux', 'Quux'],
-    ],
-    value: ['bar'],
-    onChange: (v, e) => {
-      action('MultipleCheckbox change')(v, e);
-    },
+    disabled: false,
+    name: 'multiple-checkbox-example',
   },
 };
 
-export const _MultipleCheckbox = ({...args}) => <MultipleCheckbox {...args} />;
+export const _MultipleCheckbox = ({...args}) => {
+  const [value, setValue] = useState(['bar']);
+
+  return (
+    <MultipleCheckbox
+      value={value}
+      onChange={(newValue, e) => {
+        setValue(newValue);
+        action('MultipleCheckbox change')(newValue, e);
+      }}
+      {...args}
+    >
+      <MultipleCheckbox.Item value="foo">Foo</MultipleCheckbox.Item>
```

Comment:
Is this the same way that we export things for our other compound components?

Reply:
I don't think so, but I think this pattern is good for components like this because it makes it clear that `Checkbox.Item` isn't meant to be used outside of `MultipleCheckbox`.  Also pollutes exports less so we don't have  people accidentally auto-importing the wrong `Item`.

This is a bit exploratory though. Happy to change to individual exports since that matches our other components better. I do think something like this would be good in the future if we wanted to make compound components.

Reply:
Yeah I generally agree with you.

How many other "compound" components do we have that would need to be adjusted?

Reply:
The only other example I can think of that we have already is `InputGroup`. Which actually is quite a good example because it was only added a couple months ago and someone already imported the wrong `Input`: https://github.com/getsentry/sentry/blob/49f62b8537722f38e59678b3481ad6be573ef913/static/app/components/replays/shareButton.tsx#L7

Reply:
lol yeah this is no good. Let's rope @vuluongj20 in, what do you think, should we have `InputGroup.Input` or something like that?

Reply:
Hmm Tabs is another example of the current approach:
https://github.com/getsentry/sentry/blob/master/static/app/components/tabs/index.tsx#L54

I agree with you @malwilley about how this new pattern is more explicit. I've considered using it before and chose not to because it's a bit cumbersome — to me having to prefix everything with `InputGroup.` or `Tabs.` or `MultipleCheckbox.` makes the code less easily readable.

Reply:
Maybe this would be a good topic to bring up in the next frontend TSC? It's a matter of preference and other engineers might have more thoughts/opinions on it. I don't feel too strongly either way

Reply:
Added this as an agenda item for the next TSC meeting 😃